.travis.yml

```diff
@@ -14,15 +14,19 @@ matrix:
 before_install:
   # Make sure docker image can be built
   - docker build -t dexbot/dexbot .
+  - docker pull bitshares/bitshares-core:testnet
 install:
   - pip install pyinstaller
   - pip install --upgrade setuptools
   - make install
+  - python3 -m pip install -r requirements-dev.txt
```

Comment:
Skip this install for Mac since the test are only ran for Linux? 
Also should the dev requirements be uninstalled afterwards?

Reply:
Both true, thanks.